tools/ci-build/publisher/src/subcommand/fix_manifests.rs

```diff
@@ -25,6 +25,8 @@ use tracing::{debug, info};
 
 mod validate;
 
+const AWS_SDK_CRATE_NAME_PREFIX: &str = "aws-sdk-";
```

Comment:
There is already code somewhere that categorizes package names. Shouldn't need to do this manually.

Reply:
Didn't realize there is already one existing! Fixed in bddc9e5.